README.md

```diff
@@ -132,5 +132,21 @@ yarn test
 ## Notes
 - Almost all functions are asynchronous and return promises. [You can read about promises here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_promises).
 
+## Documentation
+
+Documentation is generated using [Typedoc](https://typedoc.org/). The documentation is available at https://standardnotes.github.io/snjs/. To update the docs or to view the latest version, run:
+
+```
+yarn docs
+```
+
+This will generate documentation and place it in the `docs/` directory. Create a localhost server using `http-server` and open the [docs/index.html](docs/index.html) file in a browser to view the docs.
+
+To deploy the latest version of the docs to the `gh-pages` branch to publish using [GitHub Pages](https://pages.github.com/), replace `2.0.41` with the latest version and run:
+
+```
+yarn deploy-docs -m "build: v2.0.41"
```

Comment:
Can this be a part of the Github Actions workflow when we are releasing a new version? See: https://github.com/standardnotes/snjs/blob/master/.github/workflows/publish.yml

Reply:
Yes, it could be part of the GitHub Actions workflow when we release a new version or a separate workflow after a new push. Which do you think is better? Do you want me to figure out how to add the new workflow or do you want to do it?

Reply:
should be in the `publish` workflow right after the package is published we should add a job for publishing docs

Reply:
That sounds good

Reply:
Do you want me to update the publish workflow or do you want to do it?

Reply:
please update it if you can - if it's too troublesome I'll attend it at a later stage